system-test/maxtest/src/galera_cluster.cc

```diff
@@ -48,12 +48,21 @@ bool GaleraCluster::start_replication()
     auto gcomm = ss.str();
 
     run_on_every_backend([&](int i){
+        copy_to_node(i,
+                     (mxt::SOURCE_DIR + std::string("/mdbci/cnf/") + get_srv_cnf_filename(i)).c_str(),
```

Comment:
We have this:
```c++
void MariaDBCluster::reset_server_settings(int node)
{
    string cnf_dir = m_test_dir + "/mdbci/cnf/";
    string cnf_file = get_srv_cnf_filename(node);
    string cnf_path = cnf_dir + cnf_file;
    string home = access_homedir(node);

    // Note: This is a CentOS specific path
    ssh_node(node, "rm -rf /etc/my.cnf.d/*", true);

    copy_to_node(node, cnf_path.c_str(), home.c_str());
    ssh_node_f(node, false, "sudo install -o root -g root -m 0644 ~/%s /etc/my.cnf.d/", cnf_file.c_str());
```
Ths is called by `reset_servers()` which is called right before `start_replication()`;
```c+
            if (reset_servers())
            {
                log.log_msgf("%s reset. Starting replication.", namec);
                start_replication();
```
Why does the file have to be copied again in `start_replication()`?